guzzlehttp/guzzle/CVE-2016-5385.yaml

```diff
@@ -0,0 +1,8 @@
+title:     HTTP Proxy header vulnerability
+link:      https://github.com/guzzle/guzzle/releases/tag/6.2.1
+cve:       CVE-2016-5385
+branches:
+    master:
+        time:     2015-07-15 17:14:23
+        versions: ['>4.0.0rc2 <6.2.1']
```

Comment:
Thanks for the correction, I'll get that added. The httpoxy.org site presently has a [conflicting report](https://httpoxy.org/#php).